lib/logstash/inputs/elasticsearch.rb

```diff
@@ -271,6 +271,8 @@ class LogStash::Inputs::Elasticsearch < LogStash::Inputs::Base
 
   BUILD_FLAVOR_SERVERLESS = 'serverless'.freeze
   DEFAULT_EAV_HEADER = { "Elastic-Api-Version" => "2023-10-31" }.freeze
+  # CODEREVIEW: What should the value be here? Logstash? Logstash-plugin?
+  INTERNAL_ORIGIN_HEADER = { 'x-elastic-internal-origin' => 'logstash-input-elasticsearch'}.freeze
```

Comment:
What should the value be for plugins? Do we want it to be unique for each plugin? Do we want to indicate the request is coming from a plugin at all? Should it just be something even more generic like "logstash"?